packages/webapp/src/containers/CustomSignUp/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { useDispatch } from 'react-redux';
+import PureCustomSignUp from '../../components/CustomSignUp';
+import { customSignUp } from './saga';
+
+function CustomSignUp() {
+  const { register, handleSubmit, errors, watch } = useForm({ mode: 'onBlur' });
+  const validEmailRegex = RegExp(/^$|^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i);
+  const EMAIL = 'email';
+  const email = watch(EMAIL, undefined);
+  const required = watch(EMAIL, false);
+  const refInput = register({ pattern: /^$|^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i });
+
+  const disabled = !email || !required || !validEmailRegex.test(email);
+
+  const dispatch = useDispatch();
+
+  const onSubmit = (data) => {
+    const { email } = data;
+    dispatch(customSignUp(email));
+  };
+
+  const ssoSignUp = () => {};
```

Comment:
If we don't need this function, we can likely remove it from the props of `PureCustomSignUp`

Reply:
Done.